ibis/tests/all/test_window.py

```diff
@@ -224,8 +226,55 @@ def test_window(backend, alltypes, df, con, result_fn, expected_fn):
     backend.assert_series_equal(left, right)
 
 
+@pytest.mark.parametrize(
+    ('result_fn', 'expected_fn'),
+    [
+        param(
+            lambda t, win: t.double_col.mean().over(win),
+            lambda df: (df.double_col.expanding().mean()),
+            id='mean',
+        ),
+        param(
+            # Disabled on PySpark and Spark backends becuase in pyspark<3.0.0,
+            # Pandas UDFs are only supported on unbounded windows
+            lambda t, win: mean_udf(t.double_col).over(win),
+            lambda df: (df.double_col.expanding().mean()),
+            id='mean_udf',
+            marks=[
+                pytest.mark.udf,
+                pytest.mark.skip_backends([PySpark, Spark]),
```

Comment:
We technically could check the version of `pyspark` here and only skip if `pyspark<3.0.0`, but following in the same vein as #2364, I am avoiding adding package version checks for tests in `tests/all`. This is because `pyspark` might not even be installed here, depending on which CI job is running. I feel a version check would be more complex than its worth

Reply:
So this works for pyspark 3 but we are still skipping it?

Reply:
Oh potential way to do this is to do a try import at the top:

```
try:
    import pyspark
    _pyspark_version = ... # find out spark version
except ImportError:
     _pyspark_version = None
```

Reply:
FYI this is how spark does it:
https://github.com/apache/spark/blob/master/python/pyspark/sql/tests/test_pandas_udf.py#L24

We could have perhaps a `ibis.tests.utils` to encapsulate this

Reply:
I think this might be adding unnecessary complexity. So let's leave comment here at least and perhaps address as follow up.

Reply:
Sounds good. I have a comment already about that parameter being disabled due to `pyspark<3.0.0` but let me know if you think I should expand the comment

Reply:
No the current comment is good.